tensorboard/plugins/histogram/histograms_plugin_test.py

```diff
@@ -173,6 +175,24 @@ def test_histograms_with_histogram(self):
     self._test_histograms(self._RUN_WITH_HISTOGRAM,
                           '%s/histogram_summary' % self._HISTOGRAM_TAG)
 
+  def _test_histograms_csv(self, run_name, tag_name, should_work=True):
+    self.set_up_with_runs([self._RUN_WITH_LEGACY_SCALARS,
+                           self._RUN_WITH_SCALARS,
+                           self._RUN_WITH_HISTOGRAM])
+    if should_work:
+      (data, mime_type) = self.plugin.histograms_impl(
+        tag_name, run_name, None, histograms_plugin.OutputFormat.CSV)
```

Comment:
Rather than passing None for downsample_to, just omit it and use a keyword arg for the output format like `histograms_imp(tag_name, run_name, output_format=histograms_plugin.OutputFormat.CSV)`.